test/distributed/_tensor/test_dtensor_ops.py

```diff
@@ -670,11 +671,11 @@ def world_size(self) -> int:
     @ops(dtensor_lagging_op_db, allowed_dtypes=(torch.float,))
     @skipOps("TestDTensorOps", "test_dtensor_op_db", dtensor_fails)
     def test_dtensor_op_db(self, dtype, op):
-        pg_backend = "nccl" if DEVICE_TYPE == "cuda" else "gloo"
-        if pg_backend == "nccl" and torch.cuda.device_count() < self.world_size:
+        # pg_backend = "nccl" if DEVICE_TYPE == "cuda" else "gloo"
```

Comment:
Any reason to keep this comment?

Reply:
this PR is still WIP, might need more fixes to make it to a good shape, let me convert it to a draft